cli/cmd/test_connection.go

```diff
@@ -112,6 +112,10 @@ func testConnection(cmd *cobra.Command, args []string) error {
 	ctx := cmd.Context()
 	updateSyncTestConnectionStatus(cmd.Context(), log.Logger, cloudquery_api.SyncTestConnectionStatusStarted)
 
+	// in the cloud sync environment, we pass only the relevant environment variables to the plugin
+	isolatePluginEnvironment := apiAuth.NewTokenClient().GetTokenType() == apiAuth.SyncTestConnectionAPIKey
```

Comment:
There's a fancy `CQ_CLOUD` env you could use, see https://github.com/cloudquery/cloudquery/pull/18128/files#diff-9adce87fe0e31bd984d6a819b5bbbcca2b0f2239249ae2a06f1d502f895fb7adR35

Reply:
Maybe also update the sync command while you're at it

Reply:
Unless the `sync` command is already using that, it would be outside of the scope.

Reply:
Updated